include/vast/Dialect/HighLevel/HighLevelOps.td

```diff
@@ -1035,7 +1035,7 @@ def AddrLabelExpr
 }
 
 def InitListExpr
-  : HighLevel_Op< "initlist" >
+  : HighLevel_Op< "initlist", [NoMemoryEffect] >
```

Comment:
is that true? what if I have `{ f(), g() }`?

Reply:
@Jezurko ^

Reply:
Yeah, this should be removed. I added it at first, because without it was triggering some asserts in some later passes (but it shouldn't get there anyway).